mypy.ini

```diff
@@ -4,6 +4,23 @@ mypy_path = ./app,./app/stubs
 plugins =
     mypy_django_plugin.main
 explicit_package_bases = True
+exclude = ^.*[/\\]test_.*
+
+# add some additional checks, but not quite strict
+strict = True
+disallow_untyped_calls = False
+ignore_missing_imports = True
+implicit_reexport = true
+warn_unreachable = True
 
 [mypy.plugins.django-stubs]
 django_settings_module = "config.settings_dev"
+
+# relax rules for stubs
+[mypy-environ.*]
+disallow_untyped_defs = False
+disallow_incomplete_defs = False
+
+[mypy-gunicorn.*]
+disallow_untyped_defs = False
+disallow_incomplete_defs = False
```

Comment:
Why not the more generic `ignore_errors` (per module, not global)?

Reply:
Not sure if this is a good idea.

Reply:
I agree it looks a bit overly general. Then again: What is the advantage of having mypy report errors from an untyped package?

Reply:
You want some (minimal) checking of the stubs.

Reply:
Sure but for `gunicorn`, there are no stubs if I understand correctly. Only for `django` thanks to `django-stubs`. So maybe only set `ignore_errors` for `gunicorn`?

Reply:
https://github.com/geoadmin/service-control/tree/develop/app/stubs/gunicorn/app